gunicorn_config.py

```diff
@@ -10,8 +10,8 @@
 workers = 4
 worker_class = "eventlet"
 worker_connections = 256
-errorlog = "/home/vcap/logs/gunicorn_error.log"
-bind = "0.0.0.0:{}".format(os.getenv("PORT"))
+# errorlog = "/home/vcap/app/gunicorn_error.log"
```

Comment:
So by removing this setting, it will go back to the default for `errorlog` which appears to be `stderr` (https://docs.gunicorn.org/en/stable/settings.html#errorlog) which is what we want for ECS (because in ECS we will be sending all our logs to stdout and stderr so ECS can pick them up and ship them to cloudwatch.

So essentially, this is a good change for ECS (and I reckon you commented it out because you may ran into problems with trying to write logs to file locally when running too)

Reply:
Might be a good time to check with the rest of the dev team if they ever use the error log when debugging.

Most messages look like this:

![Screenshot 2023-02-27 at 14 02 17](https://user-images.githubusercontent.com/4289633/221559580-14226259-2094-444c-928d-9a7905ef11b9.png)

And even if we filter out the `INFO` messages and increase the time span to the last 12 weeks we see things like this:

![Screenshot 2023-02-27 at 14 05 23](https://user-images.githubusercontent.com/4289633/221559694-14a6e6a8-d306-49d6-9023-9e487c960239.png)